content/docs/user-guide/plots.md

```diff
@@ -0,0 +1,420 @@
+# Plots Configuration and Visualization
+
```

Comment:
> I made a new guide page since I thought shoving that all into the dvcyaml/project structure would be way too much

I definitely agree to put the explanations in this new page! It's a great step towards https://github.com/iterative/dvc.org/issues/2572. But ...